app/main.py

```diff
@@ -256,55 +262,48 @@ def manage_login_modal(n_clicks, username, password):
         form_feedback.append(html.P("Il semble qu'il manque votre nom d'utilisateur et/ou votre mot de passe."))
 
         # The login modal remains open; other outputs are updated with arbitrary values
-        return True, form_feedback, '', [10, 10], 3
+        return True, form_feedback, [10, 10], 3
 
     else:
-        # We open the encrypted file
-        with open(os.path.join(os.path.dirname(__file__), "data/login_correspondences.enc"), 'rb') as file:
-            enc_file = file.read()
-
-        # We decrypt the file with the password stored in the git-ignored .env file
-        csv_text = decrypt(cfg.LOGIN_CORRESPONDENCES_KEY, enc_file).decode('utf8')
-
-        # We create a temporary csv out of the decrypted string
-        correspondences = StringIO(csv_text)
 
-        # We read the resulting csv as a Pandas DataFrame
-        correspondences = pd.read_csv(correspondences)
+        login_route_url = 'http://pyronear-api.herokuapp.com/login/access-token'
 
-        if username not in correspondences['username'].values or password not in correspondences['password'].values:
-            # If either the username or the password is not found in the corresponding field of the login corresponden-
-            # ces csv file, the condition is verified
+        data = {
+            'username': username,
+            'password': password
+        }
```

Comment:
It may sounds like a super dumb question, but when/where the fetch between the submitted username and pwd and this data object occurs ?

Reply:
I am not sure to understand well the question but the username and password entered by the users are both a `State` of the callback, so they are passed as arguments to the `manage_login_modal` function. This allows to create the mini-dictionary `data` and to post it via `requests` on the next line. Is it what you wanted to know?

Reply:
Oh well did not see the State ....